test/integration/liquid/register-tags.spec.ts

```
@@ -9,18 +9,22 @@ describe('liquid#registerTag()', function () {
+  // SYNC FORK: async tag render functions are rejected at registration time
+  it('should reject async tag render at registration time', () => {
     const liquid = new Liquid()
-    liquid.registerTag('async-string', {
-      render: async () => 'B'
-    })
-    const html = await liquid.parseAndRender(`A{% async-string %}C`)
-    return expect(html).toBe('ABC')
+    // NOTE: eval is used because ts-jest compiles `async () => ...` down to a
+    // plain function at the ES6 target; eval'd code keeps the native AsyncFunction.
+    expect(() => liquid.registerTag('async-string', {
+      render: eval('(async () => "B")')
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)